src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -247,7 +249,6 @@ private static ProcessHelper CreateProcHelper(string command, string args, strin
                 ProcessWillExit = true,
                 Executable = command,
                 Arguments = args,
-                WorkingDirectory = workingDir,
```

Comment:
If you're going to remove setting it, you can remove taking it as a parameter.

Reply:
good catch!